server/tests/helpers.py

```diff
@@ -413,3 +419,175 @@ def string_to_bytes_io(string: str) -> io.BytesIO:
     string_io = io.StringIO(string)
     bytes_io = io.BytesIO(string_io.read().encode("utf-8"))
     return bytes_io
+
+
+def upload_file_helper(
+    client: FlaskClient,
+    url: str,
+    filename: str,
+    file_type: str,
+    file_content: io.BytesIO,
+    cvr_file_type: Optional[str] = None,
+):
+    rv = client.post(
+        "/api/file-upload",
+        data={
+            "file": (
+                file_content,
+                filename,
+            ),
+            "key": f"test_dir/{filename}",
+        },
+    )
+    assert_ok(rv)
+
+    return client.post(
+        f"{url}/upload-complete",
+        data={
+            "storagePathKey": f"test_dir/{filename}",
+            "fileName": filename,
+            "fileType": file_type,
+            "cvrFileType": cvr_file_type,
+        },
+    )
+
+
+def setup_ballot_manifest_upload(
```

Comment:
These helpers are great! One nit: they are doing more than just set up the upload, they are doing it, so might be slightly clearer to name them: `upload_ballot_manifest`, etc.